pkg/controller/pxcrestore/restore.go

```diff
@@ -114,8 +140,10 @@ func (r *ReconcilePerconaXtraDBClusterRestore) createJob(job *batchv1.Job) error
 			case batchv1.JobComplete:
 				return nil
 			case batchv1.JobFailed:
-				return errors.New(cond.Message)
+				errMsg = cond.Message
+				continue
```

Comment:
why don't we return error here? do we need `errMsg` variable elsewhere?

Reply:
fixed ff9d827f833ffa481255068e253239c876ddb089